### PR TITLE
Css canvas medical

### PR DESCRIPTION
**Pull request by Eugenepark215**
Screen Shot 2022-08-27 at 6 52 04 PM (2)
Screen Shot 2022-08-27 at 6 51 05 PM (2)
Screen Shot 2022-08-27 at 6 51 15 PM (2)

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great job so far. We can make some improvements though. 

1. Instead of naming all of your columns based on their order, name them based on how big they are (column-half, column-one-fourth, column-full).
2. Don't use a bunch of left margin to push things to the middle of the page. Instead, make the row have a `max-width` and a `margin: 0 auto` so it centers on the page.

We'll start there.

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-28 at 10 13 17 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Eugene! One significant note for this version: There should only be a single `row` class, not a variant class for each row you use. You should make one `row` class with only `display: flex` applied to it and make additional classes for any unique styling needed for specific elements that you can add to the element in addition to the `row` class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Please update your other `row` variant classes, as well 🙏

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Just submitting another review so I know when this is actually ready to be reviewed again 🤓

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            tasks within the platform. You stay present with patients while your</br> whole
+            care team stays up-to-speed. Be a better doctor with Canvas,<br> designed for
+            primary care providers with the help of primary care </br>providers.
+          </p>
+          <button>Get Started</button>
+        </div>`
+      </div>
+    </div>
+    <div class="row">
+      <div class="row-clinicians">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:54]: Same here, no specific `row` classes for specific sections.

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        </div>
+        <div class="column-half">
+          <p>Ward Medical Clinic, Bonifay FL</p>
+          <p>Caribou Medical, Towanda PA</p>
+          <p>Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row-ready">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:75]: Same here, no specific `row` classes for specific sections.

File: css-canvas-medical/index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          <h1>Ready to learn more?</h1>
+          <p>Schedule a private meeting now</p>
+        </div>
+        <div class="column-half">
+          <button>Get Started</button>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row-contact">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:86]: Same here, no specific `row` classes for specific sections.

File: css-canvas-medical/index.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            Product<br></br>
+            Implementation<br></br>
+            For Health Plans<br></br>
+            Security<br></br>
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="row-privacy">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:115]: Same here, no specific `row` classes for specific sections.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]
Nice work! 👍

**Pull request merged**